packages/ngtools/webpack/README.md

```diff
@@ -37,13 +45,24 @@ The loader works with webpack plugin to compile your TypeScript. It's important
+* `entryModule`. Optional if specified in `angularCompilerOptions`. The path and class name of the main application module. This follows the format `path/to/file#ClassName`.
 * `mainPath`. Optional if `entryModule` is specified. The `main.ts` file containing the bootstrap code. The plugin will use AST to determine the `entryModule`.
-* `skipCodeGeneration`. Optional, defaults to false. Disable code generation and do not refactor the code to bootstrap. This replaces `templateUrl: "string"` with `template: require("string")` (and similar for styles) to allow for webpack to properly link the resources.
+* `skipCodeGeneration`. Optional, defaults to `false`. Disable code generation and do not refactor the code to bootstrap. This replaces `templateUrl: "string"` with `template: require("string")` (and similar for styles) to allow for webpack to properly link the resources.
 * `sourceMap`. Optional. Include sourcemaps.
 * `compilerOptions`. Optional. Override options in `tsconfig.json`.
 * `contextElementDependencyConstructor`. Optional. Set to `require('webpack/lib/dependencies/ContextElementDependency')` if you are having `No module factory available for dependency type: ContextElementDependency` errors.
-* `directTemplateLoading`. Optional. Causes the plugin to load component templates (HTML) directly from the filesystem.  This is more efficient if only using the `raw-loader` to load component templates.  Do not enable this option if additional loaders are configured for component templates.
+* `directTemplateLoading`. Optional. It causes the plugin to load component templates (HTML) directly from the filesystem.  This is more efficient if only using the `raw-loader` to load component templates.  Do not enable this option if additional loaders are configured for component templates.
+* `forkTypeChecker`. Optional, defaults to `true`. Run the TypeScript type checker in a forked process.
+* `hostReplacementPaths`. Optional. It allows replacing resources with other resources in the build.
+* `platform`. Optional, defaults to `0`. Possible values are `0` and `1`. `0` stands for browser and `1` for server.
+* `logger`. Optional. A custom logger that sends information to STDOUT and STDERR.
+* `nameLazyFiles`. Optional. If `true` then uses the `[request]` placeholder to set dynamic chunk names.
+* `missingTranslation`. Optional, defaults to `warning`. Possible values are `warning`, `error` or `ignore`. Determines how to handle missing translations for i18n.
+* `i18nInFile`. Optional. Localization file to use for i18n.
+* `i18nInFormat`. Optional. The format of the localization file.
+* `i18nOutFile`. Optional. The name of the file to write extractions to.
+* `i18nOutFormat`. Optional. The format of the localization file where extractions will be written to.
+* `locale`. Optional. Locale to use for i18n.
```

Comment:
All of the i18n options should indicate that they only apply when Ivy is disabled.

Reply:
Umm.. should I add something like:

*Can be applied when Ivy is disabled*

or what would you advise?

Reply:
I think something like the below should do;
`Optional and only used for View Engine compilations.`

Reply:
@alan-agius4 Done.